apps/mobile/mobile-app/components/ui/table/styles.tsx

```
@@ -1,44 +1,44 @@
-import { tva } from '@gluestack-ui/nativewind-utils/tva';
-import { isWeb } from '@gluestack-ui/nativewind-utils/IsWeb';
+import { isWeb } from '@gluestack-ui/nativewind-utils/IsWeb'
+import { tva } from '@gluestack-ui/nativewind-utils/tva'
 
-const captionTableStyle = isWeb ? 'caption-bottom' : '';
+const captionTableStyle = isWeb ? 'caption-bottom' : ''
 
 export const tableStyle = tva({
-  base: `table border-collapse border-collapse w-[800px]`,
-});
+	base: `table border-collapse border-collapse w-[800px]`,
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)